packages/@stylexjs/postcss-plugin/__tests__/__fixtures__/.babelrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "presets": [["@babel/preset-env", { "targets": { "node": "current" } }]],
+  "plugins": [
+    ["@stylexjs/babel-plugin", { "dev": false, "runtimeInjection": false }]
+  ]
+}
```

Comment:
Make this `.babelrc.js`

Reply:
Fixed in https://github.com/facebook/stylex/pull/1067/commits/4d7ec46aaab685fbdc7193f4bf352bb8d0347230